Makefile

```diff
@@ -25,6 +25,26 @@ export DEFAULT_ROUTING ?= basic
 export KUBECONFIG ?= ${HOME}/.kube/config
 export DEVWORKSPACE_API_VERSION ?= 1f335562c475972132851c68227dd36558317bb3
 
+# By default the project should be build under GOPATH/src/github.com/<orgname>/<reponame>
+GO_PACKAGE_ORG_NAME ?= $(shell basename $$(dirname $$PWD))
+GO_PACKAGE_REPO_NAME ?= $(shell basename $$PWD)
+GO_PACKAGE_PATH ?= github.com/${GO_PACKAGE_ORG_NAME}/${GO_PACKAGE_REPO_NAME}
+
+# Set GIT_COMMIT_ID
+GIT_COMMIT_ID := $(shell git rev-parse HEAD)
+ifneq ($(shell git status --porcelain --untracked-files=no),)
+       GIT_COMMIT_ID := $(GIT_COMMIT_ID)-dirty
```

Comment:
Something, like PROW CI, don't use Makefile to build image.
What if we move this logic to Dockerfile?

Reply:
@sleshchenko that makes sense - does this look better now?

Reply:
now it should work and looks better.

Initially, I meant that we evaluate that variables during Dockerfile build, without Makefile at all. But I don't have any strong objection against reusing Makefile in Dockerfile, as far as it works fine for Dockerfile.rhel as well